src/styles/blocks/stars--0.css

```diff
@@ -0,0 +1,9 @@
+.stars--0 .stars__star:nth-child(-n + 0) {
+  background-image: url(../../images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--0 .stars__star:last-child {
+  margin-right: 0;
+}
```

Comment:
Avoid using tag names for styling elements directly as it can lead to specificity issues and is not as maintainable as using classes. Instead, consider adding a class to the '.stars__star' elements and style those classes. For the 'margin-right' property on the last child, it's good that you're targeting it specifically to avoid unnecessary right margin.